dotCMS/src/main/java/com/dotcms/content/elasticsearch/constants/ESMappingConstants.java

```diff
@@ -0,0 +1,88 @@
+package com.dotcms.content.elasticsearch.constants;
+
+/**
+ * This class define the Contentlets ESMapping constants
+ *
+ * @author oswaldogallango
+ * @version 4.2.0
+ * @since June 21, 2017
+ */
+public final class ESMappingConstants {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)